.github/workflows/test.yaml

```diff
@@ -1,50 +1,107 @@
 ---
-name: Lint & Unit Tests for S3GW UI
+name: Lint & Unit Tests for s3gw UI
 on:
   push:
     branches:
-      - '*'
+      - "*"
   pull_request:
     branches:
-      - '*'
+      - "*"
 
 jobs:
-  lint:
+  frontend-lint:
     runs-on: ubuntu-latest
 
     steps:
-
-      - name: Checkout S3GW UI
+      - name: Checkout s3gw UI
         uses: actions/checkout@v3
 
       - name: Set up node.js
         uses: actions/setup-node@v3
         with:
           node-version: 16
           cache: npm
+          cache-dependency-path: src/frontend/package-lock.json
 
       - name: Install Dependencies
-        run: npm ci
+        run: cd src/frontend && npm ci
 
       - name: Run Lint
-        run: npm run lint
+        run: cd src/frontend && npm run lint
```

Comment:
```suggestion
        working-directory: src/frontend
        run: npm run lint
```

Reply:
these will no longer be this patchset after rebasing on top of current main :)